doc/docs/2.3.x/how-tos/jupyterlab-extension/index.md

```diff
@@ -1,173 +1,115 @@
-
-You should see the repositories ready to be mounted from your Pachyderm instance.
-
-!!! Info 
-    - All mounted repositories are **read-only**. 
-    - Under the hood, the extension requires FUSE. 
-    - We apply the `/` globbing pattern to all directories/files in mounted repo@branch. 
-
-Make sure to check our [data science notebook examples](https://github.com/pachyderm/examples){target=_blank} running on Pachyderm, from a market sentiment NLP implementation using a FinBERT model to pipelines training a regression model on the Boston Housing Dataset. You will also find integration examples with open-source products, such as labeling or model serving applications. 
-
-## Install The Mount Extension
-
-The deployment instructions for Pachyderm Mount Extension come in two flavors, depending on what your deployment target is (e.g. *JupyterLab or JupyterHub on Kubernetes*).
-
-Pick the option that fits your use case:
-
-- Run with our [JupyterLab container](#running-the-jupyterlab-container).
-- Deploy on [JupyterHub with Helm](#adding-the-extension-to-your-jupyterhub-deployment-with-helm).
-
-!!! Info "Versions"
-
-     - Find the latest available version of our Pachyderm Mount Extension in [PyPi](https://pypi.org/project/jupyterlab-pachyderm/){target=_blank}.
-     - We recommend JupyterLab >= 3.0.
-<!-- Deploy the extension [locally](#on-your-machine) on a JupyterLab installed on your machine.-->
-Just before your installation... 
-### Pre-built Image vs Make Your Own
-
-Depending on your setup, you might choose to use our pre-built image containing the extension or add the extension to your image.
-
-- Using Pachyderm's pre-built image `pachyderm/notebooks-user:{{ config.jupyterlab_extension_image_tag }}`:
-
-    !!! Note 
-
-         Find the latest available tag of the image [`pachyderm/notebooks-user` in DockerHub](https://hub.docker.com/r/pachyderm/notebooks-user/tags){target=_blank} to get the latest copy of the extension.
- 
-         Our image comes with a pre-installed suite of packages, including:
+- Run and test out your pipeline code before creating a Docker image. 
 
-         - The extension `jupyterlab-pachyderm`. 
-         - A custom version of our Command-Line Tool `pachctl`. 
-         - FUSE 
-         - Additionally, it pre-creates the `/pfs` directory files that will be mounted to and grants ownership to the JupyterLab USER.
-        
-         The image is based on a GPU-enabled version of [jupyter/base-notebook¶](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-base-notebook).
+## Before You Start 
 
-- Add the extension to your existing Jupyterlab Docker image:
-
-    !!! Note 
-        Replace the following `${PACHCTL_VERSION}` with the version of `pachctl` that matches your cluster's, and update `<version>` with the release number of the extension.
+- You must have a Pachyderm cluster running.
+- You must install the Jupyterlab Helm chart
```

Comment:
Instead of "You must install the Jupyterlab helm chart" I'd say "You must add the Jupyterlab helm repository"